libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt

```diff
@@ -97,7 +104,8 @@ class CropFragment : Fragment(), UCropFragmentCallback {
                     is ImageCropAndSaveFailedState -> {
                         showCropError(state.errorMsg, state.errorResId)
                     }
-                    is ImageCropAndSaveSuccessState -> { // Do nothing
+                    is ImageCropAndSaveSuccessState -> {
+                        ImageEditor.instance.onEditorAction(CropSuccessful(state.cropResult))
```

Comment:
I think ideally all the actions would be invoked from a VM (it won't be possible for some of them, but it should hopefully work for most). I'd suggest passing ImageEditor into the `vm.start()` method so we don't access static class from a VM.

Reply:
As discussed, will try it out.